meson.build

```diff
@@ -3846,6 +3846,15 @@ public_programs += executable(
         install : true,
         install_dir : rootbindir)
 
+public_programs += executable(
+    'systemd-battery-check',
+    'src/battery-check/battery-check.c',
+    include_directories : includes,
+    link_with : [libshared],
+    dependencies : [userspace, versiondep],
+    install_rpath : rootpkglibdir,
+    install : true)
```

Comment:
indentation off. please stick to same indentation as rest of file (i.e. 8 spaces)